apps/site/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,30 @@
+import Container from "react-bootstrap/Container";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+import styles from "./NavBar.module.scss";
+
+export default function NavBar() {
+	return (
+		<Navbar expand="sm" className={`bg-body-tertiary ${styles.nav}`}>
+			<Container fluid>
+				<Navbar.Brand></Navbar.Brand>
```

Comment:
Instead of saying `<Navbar.Brand></Navbar.Brand>` you can use the more concise `<Navbar.Brand />`.

Reply:
This should be fixed by Prettier too. I'll take a closer look into it.